src/components/quote/Quote.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { H4 } from '../typography'
+import { QuoteProps } from './Quote.types'
+import { StyledBlockquote } from './Quote.styles'
+
+export const Quote: React.FC<QuoteProps> = ({ children }) => {
+  return (
+    <StyledBlockquote>
```

Comment:
When this is rendered, the DOM outputs this as `<blockquote><h4>...</h4></blockquote>`, whereas we don't wish for the `hX` to be in there 😄 

So we probably want to utilise the [`as` prop](https://styled-components.com/docs/api#as-polymorphic-prop) within the `HX` components. We want to render the `HX` (in this case, `H3`, see other comments) but `as` a `blockquote`. This means it takes the styles of `H3`, but in the DOM it appears as a `blockquote` only.

The ideal DOM output would be `<blockquote>YOUR TEXT HERE</blockquote>`.